src/modules/wallet/web3-react/connectors/Injected/IFrameEthereumProvider.ts

```diff
@@ -0,0 +1,11 @@
+import { IFrameEthereumProvider as OriginalIFrameEthereumProvider } from '@ethvault/iframe-provider'
+
+export class IFrameEthereumProvider extends OriginalIFrameEthereumProvider {
```

Comment:
for now you just extended, so we can add stuff, but we just delegate, right? in principle we could even use the original? (happy to leave it extended, just curious)

Reply:
Unfortunately, the original is not 100% relevant, it doesn't implement required `request` method